aem/build.gradle.kts

```diff
@@ -52,14 +61,20 @@ aem {
             dependsOn(":develop")
         }
 
+        environmentDev {
```

Comment:
now GAP is completely independent in case of what we have configured in Docker :)

Reply:
That gives endless opportunities :) 
What you think to organise configuration around specific containers, like httpd and knotx and configure them separately? So there is no common section like environmentDev but it is embeded under container configuration?

Reply:
>So there is no common section like environmentDev but it is embeded under container configuration?

yep